src/scripts.js

```diff
@@ -78,9 +112,6 @@ function createStartPage() {
         randomDestination1 = generateRandomDestination()
         randomDestination2 = generateRandomDestination()
         randomDestination3 = generateRandomDestination()
-        console.log(randomDestination1, '1')
```

Comment:
why are you generating random destinations? Is it simply to populate the DOM so the user can see different destination options?